terraform/development/shared/main.tf

```diff
@@ -143,6 +143,15 @@ module "database" {
   skip_final_snapshot                = true
   instance_class                     = "db.t4g.small"
 
+  scheduled_stop = {
+    enabled = true
+    timings = {
+      workday_start = "30 7"
+      workday_end   = "0 20"
+    }

```

Comment:
These timings are a bit wider than the app to give the db time to start and to have a maintenance window.